test/integration/framework/etcd.go

```diff
@@ -111,14 +130,12 @@ func RunCustomEtcd(dataDir string, customFlags []string, output io.Writer) (url
 	}
 	customURL := fmt.Sprintf("http://127.0.0.1:%d", etcdPort)
 
-	klog.Infof("starting etcd on %s", customURL)
-
 	etcdDataDir, err := os.MkdirTemp(os.TempDir(), dataDir)
 	if err != nil {
 		return "", nil, fmt.Errorf("unable to make temp etcd data dir %s: %v", dataDir, err)
 	}
-	klog.Infof("storing etcd data in: %v", etcdDataDir)
 
+	logger.V(6).Info("starting etcd", "url", customURL, "dataDir", etcdDataDir)
```

Comment:
Was URL and directory really important enough to be logged at V(0)? I don't think so. V(6) is debatable, it could also be higher.

Overall I am making the code a lot less verbose.